### DISCUSSION TITLE
Advanced logbook Tab: An error ocurred while making the request

### DISCUSSION THREAD

**Discussion by DJLax5** [magicbug/Cloudlog#1988] [Q&A] [answered]
Hi everyone! 
I am looking for some help!

I have installed cloudlog within a docker container using [this](https://hub.docker.com/r/waarlandit/cloudlog) image (I don't think that that's the issue tho). That worked fine, and I started using it over the last week. What an amazing tool!
Later I realized that my installation was not up to the current version. So I ran the update_cloudlog script, which ran without issues, and I am now tag 2.3.3.

Everything seems to work, but when I want to open the advanced logbook page I get the error message:
![image](https://user-images.githubusercontent.com/14982610/217383817-4172547c-a9b3-4f69-9c58-13f517820684.png)

I have enabled the logging in the config.php but there are absoultely no new logs after the update. Bevor the update there were logs... 
Sadly I cant tell you exactly from which point I did the update. 

Any Ideas and help would be much apprecheated! 

Thanks!

**Comment by phl0** [comment]
Could you please check the browser console to see if any errors arise during making the request?

**Comment by DJLax5** [comment, marked as answer]
Sure! 

The console displays:


`DevTools failed to load source map: Could not load content for https://clog.dm7hb.de/assets/css/bootstrap-dialog.css.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE`

`DevTools failed to load source map: Could not load content for https://clog.dm7hb.de/assets/js/bundle.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE`         

`POST https://clog.dm7hb.de/index.php/logbookadvanced/search 500  jquery-3.3.1.min.js:2`
`send @ jquery-3.3.1.min.js:2`
`ajax @ jquery-3.3.1.min.js:2`
`(anonymous) @ logbookadvanced.js?1675807075:142`
`dispatch @ jquery-3.3.1.min.js:2`
`y.handle @ jquery-3.3.1.min.js:2`
`trigger @ jquery-3.3.1.min.js:2`
`(anonymous) @ jquery-3.3.1.min.js:2`
`each @ jquery-3.3.1.min.js:2`
`each @ jquery-3.3.1.min.js:2`
`trigger @ jquery-3.3.1.min.js:2`
`w.fn.<computed> @ jquery-3.3.1.min.js:2`
`(anonymous) @ logbookadvanced.js?1675807075:367`
`l @ jquery-3.3.1.min.js:2`
`c @ jquery-3.3.1.min.js:2`
`setTimeout (async)`
`(anonymous) @ jquery-3.3.1.min.js:2`
`u @ jquery-3.3.1.min.js:2`
`fireWith @ jquery-3.3.1.min.js:2`
`fire @ jquery-3.3.1.min.js:2`
`u @ jquery-3.3.1.min.js:2`
`fireWith @ jquery-3.3.1.min.js:2`
`ready @ jquery-3.3.1.min.js:2`
`_ @ jquery-3.3.1.min.js:2 `

The first two warning messages seem to appear on any page within my cloudlog. I don't notice any css errors, seems to work fine. The Post 500 error message appears only when loading the advanced logbook. As a database I am using MariaDB 10.3.

Thanks for looking into my issue!

**Reply by AndreasK79** [reply]
What PHP version are you using?

**Reply by DJLax5** [reply]
I am using php 7.2:

`# php -v`
`PHP 7.2.24-0ubuntu0.18.04.15 (cli) (built: Nov  2 2022 09:09:52) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.2.24-0ubuntu0.18.04.15, Copyright (c) 1999-2018, by Zend Technologies`

**Reply by AndreasK79** [reply]
Upgrade to at least 7.4. That should fix your problem.

**Reply by DJLax5** [reply]
I'll try it. 
Any recommendation to what exact version I'll should upgrade? 
Just 7.4 or directly 8.0? The manual states it should be <= 8.0.
This php server only runs cloudlog

**Reply by AndreasK79** [reply]
V2 works with PHP8. PHP 8.1 might work, as we've been fixing issues here and there lately.

**Reply by DJLax5** [reply]
I just upgraded to php8.0. And yes, my issue is gone! 
Thank you so much!

**Reply by AndreasK79** [reply]
Happy to help. Enjoy!
